src/App.tsx

```diff
@@ -1,24 +1,151 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { filterTodos } from './helpers/helpers';
+import { FilterParam } from './types/FilterParam';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorType } from './types/ErrorType';
+import { Error } from './components/Error';
+import { Loader } from './Loader';
 
-const USER_ID = 0;
+const USER_ID = 6755;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [filterType, setFilterType] = useState<FilterParam>(FilterParam.All);
+  const [errorType, setErrorType] = useState(ErrorType.None);
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<null | Todo>(null);
+  const [loadingTodo, setLoadingTodo] = useState([0]);
+
+  const handleTextChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const fetchTodos = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setHasError(true);
+      setErrorType(ErrorType.Load);
+    }
+  };
+
+  const addNewTodo = useCallback((newTodo: Todo): void => {
+    setTodos((oldTodos) => [...oldTodos, newTodo]);
+  },
+  []);
+
+  const showError = useCallback((error: ErrorType) => {
+    setErrorType(error);
+    setHasError(true);
+  }, []);
+
+  const hideError = useCallback(() => {
+    setHasError(false);
+  }, []);
+
+  const removeTodo = async (todoId: any) => {
```

Comment:
Don't use any. Specify the type of todoId
![image](https://user-images.githubusercontent.com/96409563/229743479-dfa0ca91-a4de-434c-9c59-1798306b9f17.png)